bigquery/google/cloud/bigquery/table.py

```diff
@@ -76,6 +76,56 @@ def _view_use_legacy_sql_getter(table):
         return True
 
 
+class EncryptionConfiguration(object):
+    """Custom encryption configuration (e.g., Cloud KMS keys).
+
+    Args:
+        kms_key_name (str): resource ID of Cloud KMS key used for encryption
+    """
+
+    def __init__(self, **kwargs):
+        self._properties = {}
+        if 'kms_key_name' in kwargs:
```

Comment:
why not just make this an actual kwarg?

Reply:
Because I wanted it to be set to `None` if explicitly passed in as such.

Reply:
You can still do that?

Reply:
How do you tell the difference between None and unset with actual kwargs?

Reply:
You don't, my question is why do you need to?

Reply:
You're right, it shouldn't matter. It makes a difference in the JSON representation, but when it gets serialized to proto in the backend it doesn't make a difference.